pkg/types/firefly_config.go

```diff
@@ -184,4 +184,5 @@ type FireflyConfig struct {
 	Event      *EventConfig         `yaml:"event,omitempty"`
 	Plugins    *Plugins             `yaml:"plugins"`
 	Namespaces *NamespacesConfig    `yaml:"namespaces"`
+	AutoReload bool 				`json:"autoReload" yaml:"autoReload"`
```

Comment:
You only need the YAML one here

```suggestion
	AutoReload bool 				`yaml:"autoReload"`
```

Reply:
I believe this is under config so `config.autoReload` see https://github.com/hyperledger/firefly/blob/f67a468378abc7f512231ccda7db626845e0f1ec/internal/coreconfig/coreconfig.go#L130